__tests__/api/app.test.js

```diff
@@ -66,4 +65,54 @@ describe('Testing app', () => {
       });
     });
   });
+
+  describe('Endpoint /api/articles/:article_id', () => {
+    describe('Method GET', () => {
+      test('GET: 200 status', () => {
+        return request(app).get('/api/articles/1').expect(200);
+      });
+
+      test('GET: 200 status responds with an article object data', () => {
+        return request(app)
+          .get('/api/articles/1')
+          .then(({ body }) => {
+            const article = body;
+
+            const expectArticle = {
+              article: {
+                article_id: 1,
+                title: 'Living in the shadow of a great man',
+                topic: 'mitch',
+                author: 'butter_bridge',
+                body: 'I find this existence challenging',
+                created_at: '2020-07-09T20:11:00.000Z',
+                votes: 100,
+                article_img_url:
+                  'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
```

Comment:
Really good test, brilliant that you're asserting the exact article you expect the endpoint to return when given an id of 1